lib/brotli.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+var stream = require('stream')
+var inherit = require('util').inherits
+var brotliDecompressBuffer = require('brotli/decompress')
+var BrotliDecompress
+
+BrotliDecompress = function (options) {
```

Comment:
Use the constructor approach instead.

```js
var Transform = require('stream').Transform

var transformStream = new Transform({
  transform(chunk, encoding, callback) {
    // ...
  }
});
```

Reply:
Reverting back because constructor approach doesn't seem to work with browser tests